k9mail/src/main/java/com/fsck/k9/helper/MailToHelper.java

```diff
@@ -0,0 +1,157 @@
+package com.fsck.k9.helper;
+
+import android.net.MailTo;
+import android.net.Uri;
+import android.util.Log;
+
+import com.fsck.k9.K9;
+import com.fsck.k9.exceptions.ParseException;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * MailTo helper class
+ */
+public class MailToHelper {
+
+    static public final String MAILTO_SCHEME = "mailto";
+
+    // All the parsed content is added to the headers.
+    private HashMap<String, String> mHeaders;
```

Comment:
We're not using the "m" prefix for fields anymore.

Because we only support "to", "cc", "bcc", "subject" and "body" we can use final fields for each of them. That should make the resulting class immutable which is always nice because it's one less thing to worry about.